packages/gg_images/PKGBUILD

```
@@ -0,0 +1,39 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=gg_images
+pkgver=20.fe35153
+pkgrel=1
+pkgdesc='GG profile photo downloader'
+arch=('x86_64' 'aarch64')
+groups=('blackarch')
+url='https://codeberg.org/nanoory/GG_Images'
+license=('GPL3')
+makedepends=('git' 'go-pie')
```

```suggestion
makedepends=('git' 'go')
```
`go-pie` is no longer required.  ` -buildmode=pie` does the trick.

---

Got it, thanks for the info